### PR TITLE
Add changes.

### PR DESCRIPTION
**Pull request by dewniMW**

### CODE REVIEW

File: en/docs/guides/login/client-side-app.md [hunk 1]
```diff
@@ -10,34 +13,28 @@ This guide assumes you have your own application. If you wish to try out this fl
...
 Make the following requests via your application to connect your application to WSO2 IS. 
 
 1. Obtain the `authorization_code` by sending an authorization request to the authorization endpoint. 
 
     !!! tip
-        You can use [an online tool](https://tonyxu-io.github.io/pkce-generator/) to generate PKCE code challenges to include the `code challenge` and `code_challenge_method` parameters. 
-
+        You can use this [online tool](https://tonyxu-io.github.io/pkce-generator/) to generate PKCE code challenges to include the `code challenge` and `code_challenge_method` parameters. 
+        First click on `Generate Code Verifier` and then on `Generate Code Challenge`. 
+        Make note of the two values. The code challenge you get here is the base64 URL encoded value of the SHA256 hashed code_verifier so the code challenge method will be `S256`.
```
Comment (gomathyk) [inline en/docs/guides/login/client-side-app.md:36]:
```suggestion
        Note the two values. The code challenge you get here is the base64 URL encoded value of the SHA256 hashed `code_verifier` so the code challenge method will be `S256`.    
```

File: en/docs/guides/login/client-side-app.md [hunk 2]
```diff
@@ -10,34 +13,28 @@ This guide assumes you have your own application. If you wish to try out this fl
...
 
 Make the following requests via your application to connect your application to WSO2 IS. 
 
 1. Obtain the `authorization_code` by sending an authorization request to the authorization endpoint. 
 
     !!! tip
-        You can use [an online tool](https://tonyxu-io.github.io/pkce-generator/) to generate PKCE code challenges to include the `code challenge` and `code_challenge_method` parameters. 
-
+        You can use this [online tool](https://tonyxu-io.github.io/pkce-generator/) to generate PKCE code challenges to include the `code challenge` and `code_challenge_method` parameters. 
+        First click on `Generate Code Verifier` and then on `Generate Code Challenge`.
```
Comment (gomathyk) [inline en/docs/guides/login/client-side-app.md:35]:
```suggestion
        Click on **Generate Code Verifier** > **Generate Code Challenge**. 
```

File: en/docs/fragments/oauth-app-pkce.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+## OAuth/OpenID Connect configuration
+
+In the created service provider, follow the below provided steps.
```
Comment (gomathyk) [inline en/docs/fragments/oauth-app-pkce.md:3]:
```suggestion
Make the following changes to the created service provider.
```

**Review by gomathyk — APPROVED** [review]

File: en/docs/guides/login/client-side-app.md (hunk 2 above)
Comment (dewniMW) [reply en/docs/guides/login/client-side-app.md:35]: these are 2 separate buttons

**Pull request merged**